src/components/AreaDisplay/AreaDisplay.js

```diff
@@ -25,17 +26,17 @@ function AreaDisplay({onChangeRegion, region}) {
   
   function setRegion(event) {
       const region = find(regions, function(r) {return r.name === event.value});
+      setSelectedRegion(event);
       
       onChangeRegion(region);
   };
     
   return (
     <div className="AreaDisplay">
-        <p>Currently Selected Region: {region ? region.name : "none"}</p>
         <AreaSelector
             regionNames={map(regions, 'name')}
             onChange={setRegion}
-            currentRegion={region ? region.name : null}
+            currentRegion={selectedRegion ? selectedRegion : null}
```

Comment:
A small question: What is the rationale for converting a falsy value to null explicitly? Something to do with React Select? Maybe you can just pass in the value directly?

A small observation: If the conversion _is_ important, there's a [shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) for the conditional:  `selectedRegion ?? null`

Reply:
I believe I read that it has to be `null` specifically to display the "Select..." message, but it's easy to confirm that empirically, and I will do so.

Reply:
Looks like the conversion was not needed. I have simplified this, and learned something. :)